charts/retool/templates/_helpers.tpl

```diff
@@ -141,13 +141,11 @@ Usage: (include "retool.workflows.enabled" .)
 */}}
 {{- define "retool.workflows.enabled" -}}
 {{- $output := "" -}}
-{{- if or 
-    (eq (toString (default "" .Values.workflows.enabled)) "true")
-    (eq .Values.workflows.enabled true) 
-    (eq (toString (default "" .Values.workflows.enabled)) "false") 
-    (eq .Values.workflows.enabled false) 
```

Comment:
on main, with 3.7 I was getting errors like: 

```
Error: template: retool/templates/deployment_workflows_worker.yaml:1:7: executing "retool/templates/deployment_workflows_worker.yaml" at <include "retool.workflows.enabled" .>: error calling include: template: retool/templates/_helpers.tpl:146:5: executing "retool.workflows.enabled" at <eq .Values.workflows.enabled true>: error calling eq: incompatible types for comparison
```

when not explicitly setting workflows.enabled.

Reply:
no need for the default anymore?

Reply:
(this has been broken for a little while I think)

See history here... 
- https://github.com/tryretool/retool-helm/pull/120
- https://github.com/tryretool/retool-helm/pull/134
- https://github.com/tryretool/retool-helm/pull/135

https://retooled.slack.com/archives/C03THV8QH0S/p1702051476282189
-

Reply:
seems like no @anna-yn 

```
helm-3-7 template -f ~/retool-helm/charts/retool/values.yaml foo ~/retool-helm/charts/retool --set config.encryptionKey="foo" --set image.tag="3.20.16" --set codeExecutor.image.tag="3.20.16" | grep 'worker\|code-executor';
  name: foo-retool-code-executor
    retoolService: foo-retool-code-executor
# Source: retool/templates/deployment_workflows_worker.yaml
  name: foo-retool-workflow-worker
    retoolService: foo-retool-workflow-worker
            value: http://foo-retool-code-executor
  name: foo-retool-code-executor
    retoolService: foo-retool-code-executor
      retoolService: foo-retool-code-executor
        prometheus.io/job: foo-retool-code-executor
        retoolService: foo-retool-code-executor
        image: "tryretool/code-executor-service:3.20.16"
# Source: retool/templates/deployment_workflows_worker.yaml
  name: foo-retool-workflow-worker
    retoolService: foo-retool-workflow-worker
      retoolService: foo-retool-workflow-worker
        prometheus.io/job: foo-retool-workflow-worker
        retoolService: foo-retool-workflow-worker
            value: http://foo-retool-code-executor
```

in this case we are using the values.yml file in this repo, on this branch, where workflows.enabled nor codeExecutor.enabled is NOT set (blank) ... and we are using the image tags to determine if workflows should spin up or not.

Reply:
here's the same example from main branch... 

```
retool-helm git:(main) ✗ helm-3-7 template -f ~/retool-helm/charts/retool/values.yaml foo ~/retool-helm/charts/retool --set config.encryptionKey="foo" --set image.tag="3.20.16" --set codeExecutor.image.tag="3.20.16" | grep 'worker\|code-executor';
Error: template: retool/templates/deployment_workflows_worker.yaml:1:7: executing "retool/templates/deployment_workflows_worker.yaml" at <include "retool.workflows.enabled" .>: error calling include: template: retool/templates/_helpers.tpl:146:5: executing "retool.workflows.enabled" at <eq .Values.workflows.enabled true>: error calling eq: incompatible types for comparison

Use --debug flag to render out invalid YAML
```